Sources/HaCWebsiteLib/Views/BackBar.swift

```diff
@@ -0,0 +1,21 @@
+import HaCTML
+import Foundation
+
+struct BackBar: Nodeable {
+  var node: Node {
+    return El.Div[
+      Attr.className => "BackBar"
+    ].containing(
+      El.A[Attr.className => "BackBar__backButton", Attr.href => "/"].containing(
+        El.Div[Attr.className => "BigButton"].containing(
+          "Home"
```

Comment:
I'm not sure I like this as a button. Maybe we could make the `Hackers at Cambridge` logo/text clickable? Alternatively, use a back arrow? Or maybe do something to make it more 'menu-bary' although we don't currently have any other pages to add to that. 
Not sure which of these would work best though?